recipes/minikube/meta.yaml

```diff
@@ -0,0 +1,51 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Can you delete these comments from the example

Reply:
sure .. thanks! I will do it!